src/main/java/entities/Filter.java

```diff
@@ -0,0 +1,64 @@
+package entities;
```

Comment:
These Filter classes don't follow our CRC design. Could we implement Feed using the filter capability in the class? If you feel that it should be changed we can definitely discuss.